components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -0,0 +1,206 @@
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.wso2.carbon.siddhi.tryit.ui;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.siddhi.core.ExecutionPlanRuntime;
+import org.wso2.siddhi.core.SiddhiManager;
+import org.wso2.siddhi.core.event.Event;
+import org.wso2.siddhi.core.query.output.callback.QueryCallback;
+import org.wso2.siddhi.core.stream.input.InputHandler;
+import org.wso2.siddhi.core.stream.output.StreamCallback;
+
+import org.wso2.siddhi.query.api.ExecutionPlan;
+import org.wso2.siddhi.query.api.definition.AbstractDefinition;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.execution.ExecutionElement;
+import org.wso2.siddhi.query.api.execution.query.Query;
+import org.wso2.siddhi.query.compiler.SiddhiCompiler;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SiddhiTryItClient {
+
+	private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
+	private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+	private String errMsg = "";
+
+	/**
+	 * Event stream will be processed according to the specified execution plan
+	 *
+	 * @param executionPlan execution plan for siddhi query processing
+	 * @param eventStream   event stream
+	 * @param dateTime      date and time to begin the process
+	 */
+	public Map<String, StringBuilder> processData(String executionPlan, String eventStream,
+	                                              String dateTime) throws Exception {
+
+		Map<String, StringBuilder> map = new LinkedHashMap<String, StringBuilder>();
+		long beginSetTime = createTimeStamp(dateTime);
+		long beginSystemTime = System.currentTimeMillis();
+
+		// Create Siddhi Manager
+		SiddhiManager siddhiManager = new SiddhiManager();
+		ExecutionPlan newExecutionPlan = SiddhiCompiler.parse(executionPlan);
+		ExecutionPlanRuntime executionPlanRuntime =
+				siddhiManager.createExecutionPlanRuntime(executionPlan);
+
+		//Query Callback
+		for (int i = 0; i < newExecutionPlan.getExecutionElementList().size(); i++) {
+			String queryName = "";
+			ExecutionElement executionElement = newExecutionPlan.getExecutionElementList().get(i);
+
+			if(executionElement instanceof Query)   {
+				Query query = (Query) executionElement;
+				if (query.getAnnotations().size() > 0) {
+					queryName = query.getAnnotations().get(0).getElement("name");
+					if (!queryName.equals("")) {
```

Comment:
As best practise, change the equals parameters to be known/constant_variable.equals(variable)
ex: "".equals(queryName)